README.md

```diff
@@ -27,6 +27,14 @@ Start by adding Haul as a dependency to your React Native project (use `react-na
 yarn add --dev haul-cli
 ```
 
+If you're on the latest RN version, add the following in `android/app/build.gradle` somewhere before the `apply from: "../../node_modules/react-native/react.gradle"` statement:
```

Comment:
I guess it's worth specifying version number `latest` (0.43) might not be latest next week :)